static-assets/components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-assets-folder.js

```diff
@@ -554,17 +554,13 @@ var storage = CStudioAuthoring.Storage;
           var closedConfig = childIcons.closed;
           var openConfig = childIcons.open;
 
-          // adding stackedclass to folders if inProgress
-          if (treeNodeTO.statusObj.inProgress) {
-            closedConfig.icon.stackedclass = `${CStudioAuthoring.Constants.WORKFLOWICONS.edited} edited`;
-            openConfig.icon.stackedclass = `${CStudioAuthoring.Constants.WORKFLOWICONS.edited} edited`;
-          } else {
-            closedConfig.icon.stackedclass = null;
-            openConfig.icon.stackedclass = null;
-          }
+          const stackedClass = CStudioAuthoring.Utils.getContentItemWorkflowStatus(treeNodeTO);
+          closedConfig.icon.stackedclass = stackedClass;
+          openConfig.icon.stackedclass = stackedClass;
 
-          var childClosed = CStudioAuthoring.Utils.createIcon(closedConfig, '', 'on-closed');
-          var childOpen = CStudioAuthoring.Utils.createIcon(openConfig, '', 'on-open');
+          const statusClass = `${stackedClass !== 'live' ? 'cs-item-icon' : ''}`;
```

Comment:
Isn’t stackedclass empty when live?

Reply:
double checking I found that the live stackedclass does have a value